tests/src/lib.rs

```diff
@@ -119,14 +119,35 @@ pub fn mk_compiler(
     #[cfg(cargo_profile = "release")]
     let mode = "release";
 
+    let prelink_args = env::var("PRELINK_PASSES").unwrap_or_default();
+    let postlink_args = env::var("POSTLINK_PASSES").unwrap_or_default(); 
+
+    let mut yk_config_args: Vec<&str> = Vec::new();
+    yk_config_args.push(mode);
+
+    if !prelink_args.is_empty() {
+        yk_config_args.push("--prelink-pipeline");
+        yk_config_args.push(&prelink_args);
+    }
+    yk_config_args.push("--cflags");
+    yk_config_args.push("--cppflags");
+    if !postlink_args.is_empty() {
+        yk_config_args.push("postlink-pipeline");
```

Comment:
Is this supposed to be `--postlink-pipeline`?

Reply:
yes, will fix this.

Reply:
adressed here: 9e4bf0f2470deb6e2fa8cc2fd5aa2fe12d225bff